concepticondata/conceptlists.tsv

```diff
@@ -397,4 +397,5 @@ Tjuka-2021-192	Tjuka, Annika	2021		192	body parts, specific, colors	English	Glob
 Gerardi-2021-447	Gerardi, Fabrício and Aragon, Carolina and Reichert Stanislav	2021		447	basic	English,Portuguese	Tupían languages	https://calc.hypotheses.org/2988	Gerardi2021c		This list extends previous work on Tupían languages with important cultural concepts (mostly flora and fauna items).	1-6	
 Zhong-2022-664	Zhong, Yin and Wan, Mingyu and Ahrens, Kathleen and Huang, Chu-Ren	2022		664	ratings	Chinese, English	Chinese	https://osf.io/bgkyp/	Zhong2022		This list includes ratings across six perceptual modalities (touch, hearing, smell, taste, vision, and interoception) and five action effectors (mouth/throat, hand/arm, foot/leg, head excluding mouth/throat, and torso) rated by Chinese native speakers. The ratings were based on a scale from 0 (no feelings at all) to 5 (very strong feelings).		
 Araujo-1996-289	Araújo, Gabriel Antunes	1996		289	basic, documentation	German, Portuguese	Maxakalí	http://etnolinguistica.wdfiles.com/local--files/artigo%3Aaraujo-1996/araujo_1996_masakari.pdf	Araujo1996		This list, compiled and translated to Portuguese in Araújo (1996), was originally documented during fieldwork by Curt Nimuendajú in 1939, with notes published in 1958. It features basic vocabulary from Maxacalí (Nuclear-Macro-Je), a language of Brazil. It also includes some examples from the possessive paradigm. Araújo's compilation, transcriptions, and additional comments are openly available online.		
-Tjuka-2022-220	Tjuka, Annika	2022		220	body parts, specific, colors	English	Global	https://doi.org/10.5281/zenodo.6226423	Tjuka2022		The list includes 220 concepts and each concept was categorized into color, emotion or human body part. It is the second version of [Tjuka-2021-192](:ref:Tjuka-2021-192) with 28 additional emotion concepts. The list contains 22 color concepts, 62 emotion concepts, and 136 human body part concepts.		
\ No newline at end of file
+Tjuka-2022-220	Tjuka, Annika	2022		220	body parts, specific, colors	English	Global	https://doi.org/10.5281/zenodo.6226423	Tjuka2022		The list includes 220 concepts and each concept was categorized into color, emotion or human body part. It is the second version of [Tjuka-2021-192](:ref:Tjuka-2021-192) with 28 additional emotion concepts. The list contains 22 color concepts, 62 emotion concepts, and 136 human body part concepts.		
+Wang-2021-90	Wang, Xiaosha and Bi, Yanchao	2021		90	naming test	English, Chinese	Global	https://doi.org/10.1177/09567976211003877	Wang2021		This list was used as part of a psycholinguistic and neuroimaging study on inter-subjective variability of word meaning representation at the neural level. It features 90 nominal stimuli in Chinese of varying degrees of relative abstractness (independent variable), ranging from referents of high concreteness (e.g. physical objects) to higher abstractness, as in the case of logical concepts (e.g. *relationship*, *effect*), including social and emotional concepts.		
```

Comment:
Thanks for pointing out the TAG. It should be "norms, ratings" instead of "naming test".